### DISCUSSION TITLE
ONNXRuntime Issue

### DISCUSSION THREAD

**Discussion by FusionModele** [microsoft/onnxruntime#11419] [Other Q&A]
the onnx file is generated with opset 11.
I'm trying to run an onnx model with onnxruntime and I got this issue.
self.ort_session = ort.InferenceSession(model_path)
---------log------------------------
RuntimeException: [ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Exception during initialization: /onnxruntime_src/onnxruntime/core/providers/cpu/tensor/reshape_helper.h:40 onnxruntime::ReshapeHelper::ReshapeHelper(const onnxruntime::TensorShape&, onnxruntime::TensorShapeVector&, bool) gsl::narrow_cast<int64_t>(input_shape.Size()) == size was false. The input tensor cannot be reshaped to the requested shape. Input shape:{1,16074,16074}, requested shape:{1,16074,1}
----------env--------
onnxruntime-1.11.1
onnx-1.11.0

**Comment by justinchuby** [comment]
Can you share how this can be reproduced? What is your input size and does it match what the model expects?

**Reply by martin-studna** [reply]
I experienced this problem when I had CUDA 12.x and I accidently installed `sudo apt-get -y install cudnn9-cuda-13` for CUDA 13.x
